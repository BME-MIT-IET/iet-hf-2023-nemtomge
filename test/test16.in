loadmap map.json 
start V1 
set Scientist V1 ST1 
set Material ST1 nucleotide 20 
show Material V1 
touch V1 
1 
show Material V1
exit
