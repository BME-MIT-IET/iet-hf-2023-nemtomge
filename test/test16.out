Map from file map.json loaded
The game has been started. Players are: V1 
Scientist V1 set on field ST1 
Material nucleotide: 20 has been placed on field ST1
V1's materials are:
aminoacid(0) 
nucleotide(0) 
Choose one to pick:
0 - Cancel
1 - nucleotide(20)
V1's materials are:
aminoacid(0) 
nucleotide(20)
